### DISCUSSION TITLE
current power

### DISCUSSION THREAD

**Discussion by SirInsky12** [wlcrs/huawei_solar#694] [Setting up the integration]
I miss the current power consumption after the current update. Has anyone else noticed this?

**Comment by Roving-Ronin** [comment]
@SirInsky12 assume your referring to the latest HA update ?

Core                           2024.4.4
Supervisor                  2024.04.4
Operating System      12.2
Frontend                    20240404.2

with WLCRS Huawei v1.4.0a3 on two different installs and both working fine.
